### DISCUSSION TITLE
Feature request: Add music control like panel which you can see while adjusting the sound

### DISCUSSION THREAD

**Discussion by Fantoom** [File-New-Project/EarTrumpet#701] [Ideas]
I will be glad to see some music control feature in your app. I tried to search for something about it, but I only found how to add "System Media Transport Controls" in uwp app, but nothing for controlling it programmatically.

Screenshot to show what I mean

![Example screenshot](https://i.imgur.com/UTQ08fh.jpg)

**Comment by ShankarBUS** [comment]
Hey @riverar, I can work on this.

**Comment by riverar** [comment]
@ShankarBUS Thanks for the offer, but we're not looking to add this functionality at this time. And it doesn't appear Microsoft is adding this in a future release.

**Comment by ShankarBUS** [comment]
@riverar
That's fine and understandable 😊. But at least tell me how to make a plugin for EarTrumpet. So that the users who want this feature can install it optionally.

**Comment by ShankarBUS** [comment]
> And it doesn't appear Microsoft is adding this in a future release.

https://twitter.com/thebookisclosed/status/1247961816111685632?s=19

I contacted you on Twitter to tell this first
